src/components/Person.vue

```diff
@@ -34,6 +36,9 @@ export default {
     funTitle: Object,
     picture: Object,
   },
+  components: {
+    Picture,
```

Comment:
Idea: can we name this `Avatar` or `UserPicture`? For me this name is too close to HTML's native `picture` element, and it has a more specific use then being a picture (ie it's only for pictures that are people's profile picture, not other pictures, if we'll have any in this profile-centric site :-) ).